test_scripts/WebEngine/WebEngineProjection/009_OnSCU_after_Show_main.lua

```diff
@@ -0,0 +1,73 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Check that SDL transfer `OnSystemCapabilitiesUpdated` notification from HMI to an App
+-- which sent `Show` RPC with `templateConfiguration` parameter for main window after `Show` response
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- Steps:
+-- 1) App sends `Show` request with `templateConfiguration` for main window (windowID is not defined)
+-- SDL does:
+--  - proceed with `Show` request successfully
+--  - not send `OnSystemCapabilityUpdated` to App
+-- 2) HMI sends `OnSystemCapabilityUpdated` to SDL
+-- SDL does:
+--  - transfer `OnSystemCapabilityUpdated` notification to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Local Functions ]]
+local function sendShow()
+  local showParams = {
+    mainField1 = "MainField1",
+    templateConfiguration = {
+      template = "Template1"
+    }
+  }
+  local cid = common.getMobileSession():SendRPC("Show", showParams)
+  common.getHMIConnection():ExpectRequest("UI.Show")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  common.getMobileSession():ExpectNotification("OnSystemCapabilityUpdated")
+  :Times(0)
+end
+
+local function sendOnSCU()
+  local paramsToSDL = common.getOnSystemCapabilityParams()
+  paramsToSDL.appID = common.getHMIAppId()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnSystemCapabilityUpdated", paramsToSDL)
+  common.getMobileSession():ExpectNotification("OnSystemCapabilityUpdated", common.getOnSystemCapabilityParams())
+end
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update WS Server Certificate parameters in smartDeviceLink.ini file", common.commentAllCertInIniFile)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT, { appSessionId, appHMIType })
+common.Step("Start SDL, HMI", common.startWOdeviceConnect)
+common.Step("Connect WebEngine device", common.connectWebEngine, { appSessionId, config.defaultMobileAdapterType })
```

Comment:
@LitvinenkoIra  Please use "WS" value directly instead of `config.defaultMobileAdapterType` here.

Reply:
@aderiabin Done in b2655a1d4abd36817f0f99596d92e656817ac74d